dahyun1226/app/src/main/java/com/example/android_toy_project_study_2020_mvvm/recyclerview/ItemAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.android_toy_project_study_2020_mvvm.recyclerview
+
+
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+
+import android.view.ViewGroup
+
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.android_toy_project_study_2020_mvvm.R
+import com.example.android_toy_project_study_2020_mvvm.api.RepositoryDetailActivityController.Companion.intentFullName
+import com.example.android_toy_project_study_2020_mvvm.data.GithubRepoData
+import com.example.android_toy_project_study_2020_mvvm.ui.RepositoryDetailActivity
+import kotlinx.android.synthetic.main.recyclerview_item.view.*
+
+
+class ItemAdapter (private var items: ArrayList<GithubRepoData>) : RecyclerView.Adapter<ItemAdapter.ViewHolder>() {
+
+    override fun getItemCount() = items.size
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val item = items[position]
+        val listener = View.OnClickListener {
+            val intent = Intent(it.context,RepositoryDetailActivity::class.java)
+            intent.putExtra(intentFullName,item.fullName)
+            it.context.startActivity(intent)
+        }
+        holder.apply {
+            bind(listener, item)
```

Comment:
null check도 하지 않고 있기 때문에 apply가 들어갈 이유가 없습니다!
```suggestion
        holder.bind(listener, item)
```

Reply:
제거했습니다
e91a71a